freshmaker/parsers/errata/state_change.py

```diff
@@ -41,6 +42,12 @@ def parse(self, topic, msg):
         errata_id = int(inner_msg.get('errata_id'))
 
         errata = Errata()
-        return ErrataAdvisoryStateChangedEvent(
-            msg_id, ErrataAdvisory.from_advisory_id(errata, errata_id)
-        )
+        advisory = ErrataAdvisory.from_advisory_id(errata, errata_id)
+        # If advisory created by BOTAS and it's shipped,
+        # then return BotasChangeStateEvent event
+        if advisory.state == "SHIPPED_LIVE" and \
+           advisory.reporter.startswith('botas'):
+            event = BotasChangeStateEvent(msg_id, advisory)
```

Comment:
@gnaponie do you like this approach of having a separate event? Alternatively, the new handler could have a lower `order` to be processed first and return `False` in its `can_handle` method if the reporter is not BOTAS.

Reply:
I guess it was just a logic separation, also because the card says "For this handler a new type of Freshmaker event should be created".